### DISCUSSION TITLE
Does chatGPT know what is MRS?

### DISCUSSION THREAD

**Discussion by arademaker** [delph-in/docs#114] [General]
OK, just for fun and to satisfy my curiosity...

> Me: what is an MRS?

> Chat: MRS can stand for multiple things depending on the context, but one of the most common meanings is "Meaning Representation System".... 

> Me: No, MRS stands for Minimal Recursive Semantics

> Chat: You are correct. I apologize for my earlier response. In the field of computational linguistics, MRS stands for Minimal Recursion Semantics...

> Me: Can you give me the MRS representation of the sentence "the cat is white."

> Chat: 

```
[ LTOP: h1
  INDEX: e2 [ CAT: _cat_n_rel
              NUM: sg
              PER: 3
              DEF: + ]
  RELS: < [ _be_v_id_rel<0:3> LBL: h4 ARG0: e2 ARG1: e3 ]
          [ _white_a_rel<4:9> LBL: h5 ARG0: e3 ] >
  HCONS: < h6 qeq h1 > ]
```

> This MRS representation breaks the sentence down into two semantic units or relations: "_be_v_id_rel" and "_white_a_rel". "_be_v_id_rel" represents the copular verb "is" and links the subject "cat" (represented as "e2") with the predicate "white" (represented as "e3"). "_white_a_rel" represents the adjective "white" and takes "e3" as its argument. The MRS also includes other information, such as the fact that "cat" is a singular, third-person noun phrase, and that it is definite (i.e., it refers to a specific cat).

**Comment by goodmami** [comment]
The output at least looks better than when I tried it a couple months ago! It was also better at making up plausible-looking AMR than MRS. Still pretty bad, though.
